components/Collections/src/MDCCollectionViewStyling.h

```diff
@@ -142,6 +142,16 @@ typedef NS_ENUM(NSUInteger, MDCCollectionViewCellLayoutType) {
 /* Whether to hide the cell separators. Defaults to NO. */
 @property(nonatomic) BOOL shouldHideSeparators;
 
+/**
+ Returns whether the separator should be hidden for the cell with this layout attributes, determined
+ by the cell position and type.
```

Comment:
Please add a comment to mention that setting `shouldHideSeparators` to `NO` should cause this method to always return `NO`.